h/streamer/messages.py

```diff
@@ -118,7 +118,9 @@ def handle_annotation_event(message, sockets, request, session):
         return
 
     # Create a generator which has the first socket back again
-    matching_sockets = chain((first_socket,), matching_sockets)
+    matching_sockets = chain(  # pylint: disable=redefined-variable-type
```

Comment:
Yeah, not sure pytlint, chain and a generator should be pretty similar, but I guess technically different types?

Reply:
Can win 'em all